src/streams/transforms/utility/array-join-transform.ts

```diff
@@ -1,5 +1,7 @@
 import { TransformCallback, TransformOptions } from 'stream';
 import { BaseTransform } from '../base/base-transform';
+import { SimpleTransform, TransformFunction } from '../base/simple-transform';
+import { FullTransformOptions } from '../types/full-transform-options.type';
 import { TypedTransformCallback } from '../types/typed-transform-callback';
 
 export class ArrayJoinTransform<TSource> extends BaseTransform<TSource, TSource[]> {
```

Comment:
Delete due to refactor as the subsequent function fulfills this logic.

Reply:
done